Cargo.lock

```diff
@@ -1066,11 +1066,9 @@ dependencies = [
  "linkerd2-identity",
  "linkerd2-metrics",
  "linkerd2-proxy-transport",
- "linkerd2-router",
  "linkerd2-stack",
  "linkerd2-timeout",
  "rand 0.7.2",
- "regex 1.0.0",
```

Comment:
Is there a tool that you're using to prune unused deps from the Cargo.toml? If so, we might want to think about running it on CI. Not a blocker, just a thought — if we add this, it would be in a separate branch.

Reply:
Yeah, we've talked about this before. `cargo-udeps` appears to require nightly, and its output is a bit unwieldy, so i'd want to get it integrated as a proper action or something...